specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/examples/Job_CreateorUpdate_EventTrigger.json

```diff
@@ -0,0 +1,222 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "jobName": "testcontainerAppsJob0",
+    "api-version": "2023-04-01-preview",
+    "JobEnvelope": {
+      "location": "East US",
+      "properties": {
+        "environmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
+        "configuration": {
+          "replicaTimeout": 10,
+          "replicaRetryLimit": 10,
+          "eventTriggerConfig": {
+            "replicaCompletionCount": 1,
+            "parallelism": 4,
+            "scale": {
+              "minReplicas": 1,
+              "maxReplicas": 5,
+              "rules": [
+                {
+                  "name": "httpscalingrule",
+                  "custom": {
+                    "type": "http",
```

Comment:
I don't think we would support scale-type http for event driven jobs, since we don't support ingress to jobs yet. Maybe, add an example with a different scale type?

Reply:
what are the scale types we are supporting ? We should also make a change in rp then since the object for `Scale` used in rp uses all validations ....

Reply:
@tdaroly this is the spec for Scale which is being reused with jobs, I believe the validation for what types are compatible with jobs, should be done on the CP.